openquake/hazardlib/gsim/chiou_youngs_2014.py

```diff
@@ -178,7 +178,7 @@ def _get_ln_y_ref(self, rup, dists, C):
         mag_test1 = np.cosh(2. * max(rup.mag - 4.5, 0))
 
         # centered DPP
-        centered_dpp = 0.
+        centered_dpp = self._get_centered_cdpp(dists)
```

Comment:
Admitting I'm understanding correctly if you do this you'll be always computing cdpp (even when it is not needed).